### PR TITLE
Work item description view toggle added

### PR DESCRIPTION
**Pull request by sanbornsen**
View toggle feature for work item description is added. In the details page, one can toggle the description view fro Markdown to HTML.

### CODE REVIEW

File: src/app/work-item/work-item-detail/work-item-detail.component.html [hunk 1]
```diff
@@ -211,7 +211,9 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
         <!--Description-->
         <div class="separator col-md-12 col-sm-12 col-xs-12 detail-description-wrap">
           <div class="col-md-12 col-sm-12 col-xs-12 detail-description" *ngIf="loggedIn">
-            <div class="detail-desc-div-wrap" (click)="openDescription(false, true)">
+            <button class="view-switch-btn" *ngIf="descViewType=='markdown'" (click)="showHtml(desc.innerText)">View HTML</button>
```
Comment (vikram-raj) [inline src/app/work-item/work-item-detail/work-item-detail.component.html]: Can we make this button disable until there is description text?

**Comment by nimishamukherjee** [comment]
If there is no description text, and I click on it, 'undefined' is displayed.
descundefined

**Comment by nimishamukherjee** [comment]
Insert a quote behaviour:
On Git:
> a
> a
> a

In work item description:
descquote

**Comment by nimishamukherjee** [comment]
Functional test fails
descfuncerror

**Comment by sanbornsen** [comment]
@nimishamukherjee yeah that's a script injection test. We haven't decided about the current behavior of scripts in the description. Also, the code tag is not handled from the backend.

File: src/app/work-item/work-item-detail/work-item-detail.component.html (hunk 1 above)
Comment (sanbornsen) [inline src/app/work-item/work-item-detail/work-item-detail.component.html]: We can but doesn't have any use. Besides, the UX is not defined, it's just the functionality that we have implemented. Once the UX in place we'll have to rework on buttons and stuff so currently the experience is very basic and working model

**Pull request merged**